src/component/form/CaseElement.tsx

```diff
@@ -119,77 +181,108 @@ export default class CaseElement extends React.Component<CaseElementProps, CaseE
             exitName: this.props.exitName || ''
         };
 
-        this.categoryRef = this.categoryRef.bind(this);
-        this.onChangeOperator = this.onChangeOperator.bind(this);
-        this.onChangeArguments = this.onChangeArguments.bind(this);
-        this.onChangeExitName = this.onChangeExitName.bind(this);
-        this.onRemove = this.onRemove.bind(this);
-        this.validate = this.validate.bind(this);
+        bindCallbacks(this, {
+            include: [/Ref$/, /^on/, 'validate']
+        });
     }
 
     private categoryRef(ref: TextInputElement): TextInputElement {
         return (this.category = ref);
     }
 
-    private onChangeOperator(val: Operator): void {
-        if (!jsonEqual(val, this.state.operatorConfig)) {
-            const exitName = getExitName(
-                this.state.exitName,
-                val,
-                this.props.kase,
-                this.state.arguments
-            );
+    private onChangeOperator(operatorConfig: Operator): void {
+        if (!jsonEqual(operatorConfig, this.state.operatorConfig)) {
+            const updates: Partial<CaseElementState> = {
+                operatorConfig,
+                exitName: getExitName(
+                    this.state.exitName,
+                    operatorConfig,
+                    this.props.kase,
+                    this.state.arguments
+                )
+            };
+
+            if (operatorConfig.type === 'has_number_between') {
```

Comment:
we update the case's arguments array here so that the previous case's arg string (if a previous case exists) isn't passed to the `min` input of the `has_number_between` case.